collectstatic.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+mkdir -p static
```

Comment:
Create static and templates folders in project root directory if they don't exist.